src/pcapi/core/users/models.py

```diff
@@ -374,6 +387,10 @@ def eligibility_end_datetime(self) -> Optional[datetime]:
             )
         return datetime.combine(self.dateOfBirth, time(0, 0)) + relativedelta(years=constants.ELIGIBILITY_AGE_18 + 1)
 
+    @property
+    def latest_birthday(self) -> date:
+        return _get_latest_birthday(self.dateOfBirth)
```

Comment:
`User.dateOfBirth` est (malheureusement ?) un `datetime`, pas un `date` comme l'attend `_get_latest_birthday()`. Suggestion :

```suggestion
        return _get_latest_birthday(self.dateOfBirth.date)
```